### PR TITLE
Help-center: Implement happy chat with and without 3rd party cookies

### PR DESCRIPTION
**Pull request by arcangelini**
#### Changes proposed in this Pull Request

- This PR implements creating a popup for happychat inside the editor
- It also build the communication mechanism with said popup. This popups allows the chat to function where 3rd party cookies are not allowed.
- If they're allowed, we use an iframe instead of a popup, and leverage same mechanism to communicate with it.

### CODE REVIEW

File: packages/help-center/src/components/help-center-content.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { CardBody } from '@wordpress/components';
 import classnames from 'classnames';
-import { Content } from './types';
+import { Content } from '../types';
```
Comment (escapemanuele) [inline packages/help-center/src/components/help-center-content.tsx:3]:
```suggestion
import type { Content } from '../types';
```

**Review by escapemanuele — APPROVED** [review]
This is incredible. When fixing TC errors it'll be good to go! 👍

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7294847

Hi @arcangelini, could you please edit the description of this PR and add a screenshot for our translators? Ideally it'd include this string: `Unknown site`

Thank you in advance!

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7294847).

**Pull request merged**